plugins/session_notifier.rb

```diff
@@ -19,26 +22,28 @@ class SessionNotifierCommandDispatcher
       attr_reader :smtp_from
       attr_reader :minimum_ip
       attr_reader :maximum_ip
+      attr_reader :dingtalk_webhook
 
       def name
         'SessionNotifier'
       end
 
       def commands
         {
-          'set_session_smtp_address'       => 'Set the SMTP address for the session notifier',
-          'set_session_smtp_port'          => 'Set the SMTP port for the session notifier',
-          'set_session_smtp_username'      => 'Set the SMTP username',
-          'set_session_smtp_password'      => 'Set the SMTP password',
-          'set_session_smtp_from'          => 'Set the from field of SMTP',
-          'set_session_mobile_number'      => 'Set the 10-digit mobile number you want to notify',
-          'set_session_mobile_carrier'     => 'Set the mobile carrier of the phone',
-          'set_session_minimum_ip'         => 'Set the minimum session IP range you want to be notified for',
-          'set_session_maximum_ip'         => 'Set the maximum session IP range you want to be notified for',
+          'set_session_smtp_address' => 'Set the SMTP address for the session notifier',
+          'set_session_smtp_port' => 'Set the SMTP port for the session notifier',
+          'set_session_smtp_username' => 'Set the SMTP username',
+          'set_session_smtp_password' => 'Set the SMTP password',
+          'set_session_smtp_from' => 'Set the from field of SMTP',
+          'set_session_mobile_number' => 'Set the 10-digit mobile number you want to notify',
+          'set_session_mobile_carrier' => 'Set the mobile carrier of the phone',
+          'set_session_minimum_ip' => 'Set the minimum session IP range you want to be notified for',
+          'set_session_maximum_ip' => 'Set the maximum session IP range you want to be notified for',
+          'set_session_dingtalk_webhook' => 'Set the DingTalk webhook for the session notifier',
           'save_session_notifier_settings' => 'Save all the session notifier settings to framework',
-          'start_session_notifier'         => 'Start notifying sessions',
-          'stop_session_notifier'          => 'Stop notifying sessions',
-          'restart_session_notifier'       => 'Restart notifying sessions'
+          'start_session_notifier' => 'Start notifying sessions',
+          'stop_session_notifier' => 'Stop notifying sessions',
+          'restart_session_notifier' => 'Restart notifying sessions'
         }
```

Comment:
Why make these changes?  The original is much neater.

Reply:
I ran rubocop to format the file, Do you want to change to the original.

Reply:
> Why make these changes? The original is much neater.

I ran rubocop to format the file, Do you want to change to the original.

Reply:
Actually I am not very sure whether the MSF should apply the rubocop rules strictly, there are so many inconsistencies like this.  For me personally, I prefer the original code style, it's more readable.

Reply:
We are constantly updating our rubocop rules to try and keep things consistent across the codebase, it's awesome to see people running it on their PRs, thanks @cn-kali-team